libraries/chain/contracts/chain_initializer.cpp

```diff
@@ -177,7 +177,7 @@ abi_def chain_initializer::eos_contract_abi(const abi_def& eosio_system_abi)
    eos_abi.structs.emplace_back( struct_def {
       "pending_recovery", "", {
          {"account",    "name"},
-         {"request_id", "uint32"},
+         {"request_id", "sender_id_type"},
```

Comment:
If it is decided to keep using uint128_t and sender_id_type, then change all of these related sender_id params to sender_id_type, so that we won't have errors like the:

context.cancel_deferred(record["request_id"].as<sender_id_type>())

you had to fix below, since the typedef would be what was changed.